src/docs/01_introduction_and_goals.adoc

```diff
@@ -67,9 +91,13 @@ These stakeholders determine the extent and the level of detail of your work and
 Table with role names, person names, and their expectations with respect to the architecture and its documentation.
 ****
 
-[options="header",cols="1,2,2"]
+[options="header",cols="1,2"]
 |===
-|Role/Name|Contact|Expectations
-| _<Role-1>_ | _<Contact-1>_ | _<Expectation-1>_
-| _<Role-2>_ | _<Contact-2>_ | _<Expectation-2>_
+|Role/Name|Expectations
+| Developers | _Develop an application fulfilling the requierements that allow us to pass the course._
+| SOLID Developers | _Help us with our doubts concerning the SOLID specifications._
+| Course professors | _Evaluate our work and follow our progress throughout the course._
+| End users | _Be able to use our application and take advantage of the functionalities._
+| Project sponsor | __
```

Comment:
By project sponsor, we mean Tim Berners Lee. What do you think is the role of him?

Reply:
I think he wants to spead descentralized applications, so computer engineers could work together instead of depending on a big company which was the main reason to create internat (collaboration)

Reply:
You should also add clients that want to use the application

Reply:
> You should also add clients that want to use the application

By "End users" we mean clients.